src/components/site/Layout.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Nouislider from 'nouislider-react';
+import API from './API';
+import Chart from './Chart';
+import 'nouislider/distribute/nouislider.css';
+import './StyleSlider.css';
+
+class Practice extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      sliderVal1: 0,
+      sliderVal2: 0,
+    };
+  }
+
+  componentDidMount() {
+    API.get('static-data/SDSS_SpecGals_DR8_25000.json').then(res => {
+      console.log(res.data);
+      this.setState(prevState => ({
+        ...prevState,
+        data: res.data,
+      }));
+    });
+  }
+
+  onUpdate = () => value => {
+    this.setState({
+      sliderVal1: parseFloat(value[0], 10),
+      sliderVal2: parseFloat(value[1], 10),
+    });
+  };
+
+  createArray(arrObj) {
+    let max = 0;
+    let min = 1000;
```

Comment:
Should only ned to calculate the minimum and maximum values once when you get the data you need in `componentDidMount`, not here in a function you're calling every render. In `componentDidMount` set to state alongside the data, and then never touch again.